### PR TITLE
docs: translate Features page

### PR DESCRIPTION
**Pull request by juzhiyuan**
This PR aims to translate the Features page.

- [ ] Update features.json (part contents have been translated)
- [x] Update Features.tsx to load contents according to languages
- [x] Update the Features.tsx to have i18n instead of Chinese directly

### CODE REVIEW

File: data/products/api7/features.json [hunk 1]
```diff
@@ -619,42 +1288,5 @@
...
-          {
-            "title": "",
-            "href": ""
-          }
-        ]
-      }
-    ]}
   ]
-}
+}
```
Comment (juzhiyuan) [inline data/products/api7/features.json:1292]:
```suggestion
}

```

File: views/Features/Features.tsx [hunk 2]
```diff
@@ -98,8 +98,7 @@ const Features: NextPage<Props, any> = ({ t, list = [] }) => {
 Features.getInitialProps = async (context) => {
   const { lng = "zh-CN" } = (context.req as any) || {};
 
-  // After adding the English version replace "zh-CN" with lng in line 84
-  const posts = features["zh-CN"];
+  const posts = features[lng];
```
Comment (juzhiyuan) [inline views/Features/Features.tsx:101]: cc @guoqqqi to update this page's contents (i18n support)

Comment (guoqqqi) [reply views/Features/Features.tsx:101]: Do I raise a new PR to change it or do I change it in that branch?

Comment (juzhiyuan) [reply views/Features/Features.tsx:101]: in this PR is ok

File: public/static/locales/en-US/features.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "features-title-decs": "API7 provides you with a production-ready, seven-tier, full traffic processing platform"
```
Comment (juzhiyuan) [inline public/static/locales/en-US/features.json:2]:
```suggestion
  "features-title-decs": "API7 is a Production-Ready, Layer 7 traffic processing platform"
```

**Pull request closed without merging**